src/onCLS.ts

```diff
@@ -22,7 +22,15 @@ import {doubleRAF} from './lib/doubleRAF.js';
 import {onHidden} from './lib/onHidden.js';
 import {runOnce} from './lib/runOnce.js';
 import {onFCP} from './onFCP.js';
-import {CLSMetric, CLSReportCallback, ReportOpts} from './types.js';
+import {
+  CLSMetric,
+  CLSReportCallback,
+  MetricThresholds,
+  ReportOpts,
+} from './types.js';
+
+/** Thresholds for CLS. See https://web.dev/cls/#what-is-a-good-cls-score */
+export const CLSThresholds: MetricThresholds = [0.1, 0.25];
```

Comment:
FWIW exporting like this means that users will be able to change the thresholds used, which is likely not a use case we want to get into supporting (maybe it'll be rare if ever, but, https://xkcd.com/1172/).

Could copy or export a function that returns a copy.

It may be worth returning a different format at that point, e.g. an object with named properties could be more self explanatory than `[number, number]`.

Reply:
> FWIW exporting like this means that users will be able to change the thresholds used, which is likely not a use case we want to get into supporting (maybe it'll be rare if ever, but, https://xkcd.com/1172/).

Lol, good call. I'll update the exported thresholds to prevent changes.

- [ ] Disallow changes to internal threshold values via exports

> It may be worth making the external version a different format at that point, e.g. an object with named properties could be more self explanatory than [number, number].

I agree it would be better for the format to be more self-explanatory. (Although @tunetheweb [had some concerns](https://github.com/GoogleChrome/web-vitals/pull/323#issuecomment-1446277301) about increased build file sizes.) How about something like this? 

```ts
/**
 * The ranges defining a metric's "good", "needs improvement", and "poor"
 * thresholds:
 *
 * - Metric values ≦ good are "good"
 * - Metric values > good and ≦ needsImprovement are "needs improvement"
 * - Metric values > needsImprovement are "poor".
 */
export type MetricThresholds = {
  good: number;
  needsImprovement: number;
};
```

Reply:
I wonder if we should instead have a general `getMetricThresholds()` function exported that would prevent changing of the values and also allow only a single extra export? For example, `getMetricThresholds('LCP')`, or even `getMetricThresholds('LCP','good')`. That may require moving all the thresholds to that common function and referencing it from each `onXXX` function.

Then again, I'm wondering if this change is now worth the effort given it's not quite a simple as originally thought?

Reply:
I think a metric rating threshold getter function makes sense and solves both problems. 

I added a new [`getMetricRatingThresholds()` function](https://github.com/GoogleChrome/web-vitals/pull/323/files#diff-562e9b968ecef139d0bc5134e0b9ba0efad5c52017c1ddbf3419ef6864cf0a4fR43) which takes a metric name and returns a [`MetricRatingThresholds` type](https://github.com/GoogleChrome/web-vitals/pull/323/files#diff-c8915aba955333a21cf5d5713fd07a6afbd7c06aeb0e4f97bfac15618fcabdd4R110) (updated to the above proposed format). Also moved all the [threshold values](https://github.com/GoogleChrome/web-vitals/pull/323/files#diff-562e9b968ecef139d0bc5134e0b9ba0efad5c52017c1ddbf3419ef6864cf0a4fR4) alongside `getMetricRatingThresholds()` in [a new module](https://github.com/GoogleChrome/web-vitals/pull/323/files#diff-562e9b968ecef139d0bc5134e0b9ba0efad5c52017c1ddbf3419ef6864cf0a4f). 

WDYT?